spec/models/null_github_organization_spec.rb

```diff
@@ -1,11 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe NullGitHubOrganization do
+  it_behaves_like 'a NullGitHubResource descendant with attributes'
+
   subject { described_class.new }
 
   describe '#html_url' do
     it 'returns #' do
-      expect(subject.html_url).to eql('#')
+      expect(subject.html_url).to eql('https://github.com/ghost')
```

Comment:
changing this url will mean that links in classroom will go to @ghost. Do we want that?